internal/handlers/pg/msg_serverstatus.go

```diff
@@ -51,7 +40,8 @@ func (h *Handler) MsgServerStatus(ctx context.Context, msg *wire.OpMsg) (*wire.O
 
 	uptime := time.Since(h.startTime)
 
-	stats, err := h.pgPool.SchemaStats(ctx, db, "")
+	// TODO it needs to be calculated for all schemas
```

Comment:
Shouldn't we create an issue for that?

Reply:
We shouldn't because it's already implemented in this ticket :) This `TODO` is not needed, I was writing it for myself and forgot to delete :D

Reply:
Fixed